pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -452,7 +452,7 @@
     // Replicator producer queue size;
     private int replicationProducerQueueSize = 1000;
     // Enable TLS when talking with other clusters to replicate messages
-    private boolean replicationTlsEnabled = false;
+    private boolean brokerClientTlsEnabled = false;
```

Comment:
We cannot just remove the old config key or we'd break compatibility. 

We need to keep it and mark it as `@Deprecated`